db/migrate/20170310182013_add_fingerprint2_to_schemas.rb

```diff
@@ -0,0 +1,9 @@
+class AddFingerprint2ToSchemas < ActiveRecord::Migration[5.0]
+  def change
+    add_column(:schemas, :fingerprint2, :string, null: true)
+    add_index(:schemas, :fingerprint2, unique: true)
+
+    remove_index(:schemas, :fingerprint)
+    add_index(:schemas, :fingerprint, unique: false)
```

Comment:
Why remove the unique index on `fingerprint`? PG will just ignore it if it's null anyway.